apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java

```diff
@@ -163,21 +163,76 @@
+
+  /**
+   * The standard {@link Coding#getDisplay()} for {@link Identifier#getType()} entries where the
+   * identifier is a NPI.
+   */
+  static final String CODED_IDENTIFIER_TYPE_NPI_DISPLAY = "National Provider Identifier";
+
+  /**
+   * The standard {@link Coding#getCode()} for {@link Identifier#getType()} entries where the
+   * identifier is a UPIN.
+   */
+  static final String CODED_IDENTIFIER_TYPE_UPIN = "UPIN";
+
+  /**
+   * The standard {@link Coding#getDisplay()} for {@link Identifier#getType()} entries where the
+   * identifier is a UPIN.
+   */
+  static final String CODED_IDENTIFIER_TYPE_UPIN_DISPLAY = "Unique Physician Identification Number";
+
   /** The Code System, as available at <a */
-  static final String FHIR_IDENTIFIER_TYPE_CODESYSTEM =
+  static final String CODING_SYSTEM_HL7_IDENTIFIER_TYPE =
       "http://terminology.hl7.org/CodeSystem/v2-0203";
 
-  /** The Code System, as available for UPIN */
-  static final String CODING_UPIN = FHIR_IDENTIFIER_TYPE_CODESYSTEM;
+  /**
+   * Used as the {@link Coding#getSystem()} for custom-to-BFD {@link Identifier#getType()} codes.
+   */
+  static final String CODING_SYSTEM_IDENTIFIER_TYPE =
+      BASE_URL_BBAPI_RESOURCES + "/codesystem/identifier-type";
 
-  /** The Code System, as available for NCPDP */
-  static final String CODING_NCPDP = "http://ncpdp.org/provider-id";
+  /**
+   * The standard {@link Coding#getCode()} for {@link Identifier#getType()} entries where the
+   * identifier is a PDP.
+   */
+  static final String CODED_IDENTIFIER_TYPE_PDP = "NCPDP";
+
+  /**
+   * The standard {@link Coding#getDisplay()} for {@link Identifier#getType()} entries where the
+   * identifier is a PDP.
+   */
+  static final String CODED_IDENTIFIER_TYPE_PDP_DISPLAY =
+      "National Council for Prescription Drug Programs";
+
+  /**
+   * The standard {@link Coding#getCode()} for {@link Identifier#getType()} entries where the
+   * identifier is a DL.
+   */
+  static final String CODED_IDENTIFIER_TYPE_DL = "DL";
 
-  /** The Code System, as available for State License Number */
-  static final String CODING_STATE_LICENSE = FHIR_IDENTIFIER_TYPE_CODESYSTEM;
+  /**
+   * The standard {@link Coding#getDisplay()} for {@link Identifier#getType()} entries where the
+   * identifier is a DL.
+   */
+  static final String CODED_IDENTIFIER_TYPE_DL_DISPLAY = "State license number";
 
-  /** The Code System, as available for Federal Tax Number */
-  static final String CODING_FEDERAL_TAX_NUM = FHIR_IDENTIFIER_TYPE_CODESYSTEM;
+  /**
+   * The standard {@link Coding#getCode()} for {@link Identifier#getType()} entries where the
+   * identifier is a TAX.
+   */
+  static final String CODED_IDENTIFIER_TYPE_TAX = "TAX";
+
+  /**
+   * The standard {@link Coding#getDisplay()} for {@link Identifier#getType()} entries where the
+   * identifier is a TAX.
+   */
+  static final String CODED_IDENTIFIER_TYPE_TAX_DISPLAY = "Federal tax number";
```

Comment:
Yeah, I thought about suggesting you move the constants here, but wasn't 💯  on it. On the one hand, it's nice to have all the custom codesets in one place to inspect them. On the other, I'm not a fan of the indirection it adds.

Anywhoways, entirely your call on what the better choice is: here or the enum. (I guess if you wanted to put them in the enum you could drop a comment here pointing to it. 🤷 )